src/Entity/Conference.php

```diff
@@ -127,6 +127,11 @@ class Conference
      */
     private ?string $city = null;
 
+    /**
+     * @ORM\Column(type="boolean", nullable=true)
+     */
+    private ?bool $online = false;
```

Comment:
No need for `?bool` since you initialize the value + It does not make sens to set null where.
Same in the setter.